.github/workflows/codecoverage.yml

```diff
@@ -2,8 +2,6 @@
 name: "Code coverage"
 
 on:  # yamllint disable-line rule:truthy
-  push:
```

Comment:
Can you please help me understand why this was removed? I see it present in the networking collections codecoverage.yml.

Reply:
Ohh yes, i remember adding this so it runs the codecov after it was merged. This must not be removed. My bad, i confused this for a different thing.

Reply:
yep it was due to while rebasing and addressing the merge conflict, reverted it, my bad